packages/react-scripts/template/README.md

```diff
@@ -1819,6 +1820,66 @@ Learn more about React Styleguidist:
 * [GitHub Repo](https://github.com/styleguidist/react-styleguidist)
 * [Documentation](https://react-styleguidist.js.org/docs/getting-started.html)
 
+## Sharing Components in a Monorepo
+A typical monorepo folder structure looks like this:
```

Comment:
This should be guarded by something like “This feature is only available in ...” like other similar sections.